actors/miner/src/deadline_state.rs

```diff
@@ -156,19 +156,19 @@ pub struct Deadline {
     //
     // These proofs may be disputed via DisputeWindowedPoSt. Successfully
     // disputed window PoSts are removed from the snapshot.
-    optimistic_post_submissions_snapshot: Cid,
+    pub optimistic_post_submissions_snapshot: Cid,
```

Comment:
Nit: `pub` the `partitions_snapshot` too.

Reply:
sure.

Reply:
done.